rust/stracciatella/src/schemas/yaml/types/sector-id.schema.yaml

```diff
@@ -0,0 +1,4 @@
+type: string
+title: Sector ID
+description: String representation of sector ids, such as `A1`, `A2`, `B1` etc.
+pattern: ^[A-Z][0-9]+$
```

Comment:
Should we use a stricter regex? The valid range is only `A1` - `P16`. 

I mean like `^[A-P][1-9][0-6]?$` or even `^[A-P]([1-9]|1[0-6])$` but the latter might be unnecessarily overly complicated.

Reply:
Makes sense. Added the strictest one.